packages/atoms/src/NumPadButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback } from 'react';
+import { Pressable } from 'react-native';
+
+import { Text } from './Text';
+import { useNativeStyles, NativeStyleObject, prepareNativeStyle } from '@trezor/styles';
+
+export interface NumPadButtonProps {
+    value: number;
+    onPress: (value: number) => void;
+    style?: NativeStyleObject;
+}
+
+const BUTTON_SIZE = 48;
+
+export const numPadButtonStyle = prepareNativeStyle(utils => ({
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: BUTTON_SIZE,
+    height: BUTTON_SIZE,
+    borderWidth: 1,
+    borderColor: 'black',
```

Comment:
never use colors directly, use colors from utils

Reply:
Good catch! There shouldn't be border at all, I just used it for testing purposes when centering the number and forgot to remove it.